storage/src/backend/connection.rs

```diff
@@ -626,14 +626,19 @@ impl Connection {
         } else {
             None
         };
+        // get pool size from envvar
+        let pool_max_idle_per_host = match env::var("REGISTRY_CLIENT_POOL_MAX_IDLE_PER_HOST") {
```

Comment:
Do these idle connections take pressure on the registry server? Especially when the container is started and prefetch work is done. Maybe we can make the value as the one option of ConnectionConfig.

Reply:
I don't think it would make a significant impact for reasonable values. Most servers should close idle connections if it becomes a problem.

Reply:
How about making the value as the one option of ConnectionConfig?

Reply:
It's better to fetch `REGISTRY_CLIENT_POOL_MAX_IDLE_PER_HOST`  from configuration.